dafni_cli/models/upload.py

```diff
@@ -35,6 +35,10 @@ def upload_model(
         json (bool): Whether to print the raw json returned by the DAFNI API
     """
 
+    if not is_valid_definition_file(definition_path):
+        click.echo("Wrong definition file type.")
```

Comment:
This message should maybe be a bit more decriptive
```suggestion
        click.echo("Your model definition file type is incorrect. Please check you've entered the correct file and try again")
```